lir/lrsystems/bootstraps.py

```diff
@@ -97,3 +97,118 @@ def fit_transform(self, instances: FeatureData) -> LLRData:
         return LLRData: The transformed feature data with best estimate and confidence intervals.
         """
         return self.fit(instances).transform(instances)
+
+
+class BootstrapEquidistant(Pipeline):
+    """Bootstrap system that estimates confidence intervals around the best estimate of a pipeline.
+    This bootstrap system creates bootstrap samples from the training data, fits the pipeline on each sample,
+    and then computes confidence intervals for the pipeline outputs based on the variability across the bootstrap
+    samples.
```

Comment:
Uit deze beschrijving blijkt niet heel duidelijk wat het verschil is met de andere bootstrapmethode. Daarnaast is het denk ik goed om de beperking van deze aan te geven, naamelijk dat het alleen werkt op 1-dimensionale data.